.github/scripts/docker_build.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
can you put this file in selfdrive/test?

Reply:
done